src/js/ui/eac/index.tsx

```diff
@@ -0,0 +1,94 @@
+import { header as headerWidget , menuButton} from '../shared/common';
+import redraw from '../../utils/redraw';
+import * as helper from '../helper';
+import formWidgets from '../shared/form';
+import layout from '../layout';
+import i18n, { loadFromSettings, getAvailableLanguages } from '../../i18n';
+import settings from '../../settings';
+import * as m from 'mithril';
+import * as xhr from '../../xhr';
+import * as stream from 'mithril/stream';
+import * as utils from '../../utils';
+import router from '../../router';
+import { Pos } from '../../lichess/interfaces';
+export default {
+  oncreate: helper.viewSlideIn,
+
+  oninit: function() {
+    this.positions = stream([]);
+    this.endgamesPositions = stream([]);
+    this.position = '';
+    this.start = start;
+
+
+    utils.loadLocalJsonFile('data/endgames.json')
+      .then(data => {
+        this.endgamesPositions(data);
+        redraw();
+      });
+
+
+
+      function start() {
+        settings.gameSetup.ai.level('8');
+        settings.gameSetup.ai.variant('3');
+        settings.gameSetup.ai.color('white');
+        settings.gameSetup.ai.availableVariants[1];
+        
+        function startAIGame(position:string) {
+          return xhr.newAiGame(position)
```

Comment:
It has to be an offline game. So instead of making an xhr, it should use instead the `AiRound` module.

Either it redirects to ai screen (`router.set('/ai/fen/fenstring')`) or better it loads directly the `AiRound` class (which needs to be refactored for that purpose, and not be coupled to global settings anymore).